validation-test/Driver/coverage.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: rm -rf %t && mkdir %t
+
+// RUN: %target-build-swift %s -profile-generate -o %t/main
```

Comment:
This tells the compiler to emit counter updates. To check whether code coverage mappings are emitted, I suggest adding a separate run line + FileCheck for `-profile-generate -profile-coverage-mapping`.